libraries/AP_Motors/AP_MotorsHeli_Swash.cpp

```diff
@@ -0,0 +1,183 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdlib.h>
+#include <AP_HAL/AP_HAL.h>
+
+#include "AP_MotorsHeli_Swash.h"
+
+extern const AP_HAL::HAL& hal;
+
+const AP_Param::GroupInfo SwashInt16Param::var_info[] = {
+
+    // @Param: ENABLE
+    // @DisplayName: Enable settings for H3
+    // @Description: Automatically set when H3 swash type is selected. Should not be set manually.
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Advanced
+    AP_GROUPINFO_FLAGS("ENABLE", 1, SwashInt16Param, enable, 0, AP_PARAM_FLAG_ENABLE),
+
+    // @Param: SV1_POS
+    // @DisplayName: servo1_pos
```

Comment:
Maybe a better DisplayName for SV1_POS (and SV2_POS, SV3_POS)?  This looks a bit too much like a variable name I think.